### DISCUSSION TITLE
用AI寫歌？

### DISCUSSION THREAD

**Discussion by tshsu04224** [1101-GS4719-C/Python-Community-GS4719-C#2017] [Ideas]
近年來AI發展迅速，生成照片已經不稀奇了，最近一款名為suno的AI音樂製作平台引起了關注，只要簡單的操作，輸入關鍵字、曲風等基本內容，AI就可以快速幫你生成一首首歌。他還支援多種語言，除了純音樂，更可以生成有人聲的高品質歌曲，未來人人都可以享受創作音樂的樂趣。
但隨著這些AI技術的發展，連帶著一些問題的產生，像是會不會連作曲家都面臨失業，或者利用AI創作出的音樂，該歌曲又應該歸誰所有，都是新科技所帶來的隱憂。
[suno網址](https://suno.com/)

**Comment by 710388** [comment]
好像很多歌曲都是這樣，尤其是網路上的歌曲，很多都是拿以前的架構然後用ai創好幾種然後一直發，看哪個會中

**Comment by NcuJerryKao** [comment]
AI技術在音樂領域的應用確實帶來了許多新機遇和挑戰。隨著AI音樂製作平台如suno的出現，使得音樂創作變得更加便捷和普及，同時也引發了一些問題和隱憂。以下是一些建議回應：
1. 尊重和保護作曲家的權益：雖然AI技術可以幫助生成音樂，但作曲家的創意和獨特性仍然是無法取代的。應該尊重作曲家的權益，並確保其在音樂創作領域的地位和利益。
2. 建立適當的法律和倫理準則：需要建立相應的法律和倫理準則，來規範AI生成音樂的權益歸屬、著作權問題等，確保音樂創作的公平和合法性。
3. 鼓勵創新和協作：AI技術可以作為音樂創作的輔助工具，而非取代作曲家。鼓勵創新和協作，讓AI和人類共同創作出更具價值和意義的音樂作品。
4. 教育和培訓：加強對音樂創作和AI技術的教育和培訓，幫助人們更好地理解和應用AI技術，並發揮其在音樂創作上的潛力。
透過以上建議，可以更好地應對AI音樂創作所帶來的挑戰，實現AI技術和人類創意的有效結合，促進音樂創作領域的健康發展。

**Reply by yubo0oo** [reply]
貼爽了沒烙葛

**Comment by lilishh** [comment]
現階段AI能生成的音樂，感覺就會是很芭樂的抖音歌，和專門的作曲家相比應該很明顯。就目前的AI繪畫技術來看，其實還是遠不及藝術家。

**Comment by 155Julia** [comment]
我有看過、聽過Ai的美術及音樂作品，雖然確實很厲害，有人真的靠Ai畫的畫兒得獎，但我認為Ai還是無法完全取代人類藝術家，可能頂多只能參考一下創意的部分。

**Comment by ds0w0** [comment]
我自己用過Suno，雖然目前能創作的內容一定程度受到其學習的素材限制，但未來AI一定會不斷進步。AI確實讓音樂創作變得門檻更低，但我認為音樂產業不會因為新技術而沒落。隨著AI進步，固然部分音樂製作者會受到波及，但歷史反覆證明新的技術反而會創造更多機會。我認為人類藝術家應該專注在AI無法取代的部分，例如建立與人的連結、創造有趣的體驗、做更多決策、整合和承擔風險的工作等提高自身價值，這樣至少不會成為第一批被AI淘汰的人。

**Reply by zoo100130** [reply]
之前試用suno過，感覺如果單純為了生產作用或商業用音樂還可以，如果是要做成普羅大眾都想聽的藝術性可能就不夠，但確實好用: )

**Comment by CtyceT** [comment]
過去也有因為照相機的發明而人類誕生了超現實主義的藝術文化，現如今Suno的出現可能也會為藝術帶來變革，畢竟藝術確實門檻變親民了。不過我相信藝術的上限是只能讓人類創造的，科技發展和藝術碰撞出的火花很值得期待。

**Comment by slps101023** [comment]
我覺得用AI創作歌曲目前部會使創作著失業，如同久石讓(日本作曲家)採訪時所說[https://www.cna.com.tw/news/acul/202409150166.aspx](url)現在的AI只會模仿，而非創作所需要的創意。

**Comment by poyen0806** [comment]
用AI寫歌是可行的，雖然可能會導致一些原作曲家失業，但在其他面向來看的話，用AI寫歌有它的好處，例如完成一些作曲家未能完成的作品，如第十號交響曲。

**Comment by Jing-Asteroid** [comment]
有些人會用這些技術再把藝人的聲音拿去唱別人的歌，也不失為一個樂趣。但就是也會出現抄襲爭議或是歌手本人覺得不被尊重。

**Comment by CeBru13** [comment]
有些畫畫也是這樣有些甚至畫得很好

**Comment by lai1008** [comment]
有些人會用這些技術將自己喜歡的藝人的聲音拿去唱其他的歌，也不失為一個樂趣。雖然不涉及違法，但就是也會出現抄襲爭議或是歌手本人覺得不被尊重。

**Comment by 112121py** [comment]
至少現今AI還不能完全取代人類的創作，包含我們在作品中的情感、經歷、故事等等，此外AI創作其實某方面也滿有趣的，有些觀眾會把實況主的聲音合成特定的歌曲，以達到玩梗或做效果的目的，當然也會直播帶來蠻有趣的迴響，不過就像上面幾位同學說的，如果被拿來合成的聲音主人感到不適，有表達不滿的情況，就不該再惡搞，要尊重他人~

**Comment by zhilin93517** [comment]
我大二的時候有一個同學有用過AI寫歌，他雖然用很沒有營養的內容喂給AI，但AI仍然煞有其事的做了一首歌給他。這點倒是令我感到蠻佩服的，如果是我的話我都不知道那樣的歌詞要怎麼做成曲

**Comment by qiu0109** [comment]
我覺得AI寫歌適合用在以非營利為目的的地方，例如學生製作的專案報告背景音樂等，否則要找到適合的音樂並不容易，AI可以製作出較符合我們需求的音樂。但我認為還是要尊重現代的創作歌手，畢竟AI的素材庫也都是由這些人所提供，若是沒有人類不斷提供新的創意，AI所做出來的歌曲也只會是舊的曲風，毫無新意。

**Comment by Paul921103** [comment]
AI技術日新月異，像Suno這樣的工具確實帶來了很多方便與樂趣，讓一般人也能參與到音樂創作中。但在法律層面，AI生成的作品該如何歸屬卻是個問題。根據現行法律，著作權只有在人類創作的前提下才能成立，或是會歸屬於操作AI的人。如果AI生成的作品數量過多，且這些作品均由操作AI的人申請著作權，可能會對人類創作造成壓力，擠占創作者的創作空間。因此AI作品的湧現，是否會造成創作壟斷，讓人類創作的空間變小，仍需進一步討論和法律規範。
